src/python/WMCore/WMSpec/Steps/Executor.py

```diff
@@ -19,6 +19,18 @@
 getStepName = lambda step: WMStepHelper(step).name()
 getStepErrorDestination = lambda step: WMStepHelper(step).getErrorDestinationStep()
 
+#TODO this function can me moved to Utils.
```

Comment:
Yeah, that would have been my comment, a function like this really belongs into Utils.